doc/itkTextureFeature.tex

```diff
@@ -49,7 +49,7 @@
 
 % At minimum, give your name and an email address.  You can include a
 % snail-mail address if you like.
-\author{Jean-Baptiste Vimort$^{1}$, Matthew McCormick$^{1}$ François Budin$^{1}$ and Beatriz Paniagua$^{1}$}
+\author{Jean-Baptiste Vimort$^{1}$, Matthew McCormick$^{1}$ Francois Budin$^{1}$ and Beatriz Paniagua$^{1}$}
```

Comment:
@dzenanz Thanks for the pointer!

I tested it, and it worked -- I will update the branch.